homeassistant/components/zha/light.py

```diff
@@ -711,7 +711,7 @@ async def async_update(self) -> None:
             effects_count = Counter(itertools.chain(all_effects))
             self._effect = effects_count.most_common(1)[0][0]
 
-        self._attr_color_mode = None
```

Comment:
Let's not do it this way. We should use this and remove our property that overrides the base one.